### PR TITLE
Prevent resizing/portrait mode on android/iOS systems

### PR DESCRIPTION
**Pull request by Endaris**
In Love 11.4 the default behaviour of Android systems was changed as you had to use hacky solutions to get your game work in portrait mode: https://github.com/love2d/love-android/issues/196

As there is very little point in using panel in portrait mode and at Poke's complaints, I reverted to the 11.3 behaviour in this PR.

I chose a slightly "awkward" implementation, saving the OS string inside a global within the auto_updater as that would be a necessary step for the soon(tm) android storage migration anyway. It also prevents the app from spinning twice on startup which happens if resizability is changed later on via `love.window.setMode`.

### CODE REVIEW

**Comment by Endaris** [comment]
This solves the issue on my device but neither for Poke nor Ken and Kitti even reported that it even spins into portrait on 11.3.
Very inconsistent it seems.

**Comment by Endaris** [comment]
According to Kudo, this does work if the game data is wiped previously - otherwise the game will apparently try and download the new version and directly overwrite itself before the thing can be tested.
TL;DR: Kudo managed to get it working once so I think it might be worthwhile to merge this as is and see if it works properly if the clients get updated through the appropriate channel instead.

**Comment by Endaris** [comment]
Since I found out that we realistically have no way of updating the auto updater, I'll have to do this differently than here.
